Bio/PDB/MMTF/__init__.py

```diff
@@ -0,0 +1,24 @@
+from mmtf import fetch,parse
+from Bio.PDB.MMTF.DefaultParser import StructureDecoder
+
+
+def get_from_decoded(decoder):
+    structure_decoder = StructureDecoder()
+    decoder.pass_data_on(structure_decoder)
+    return structure_decoder.structure_bulder.get_structure()
+
+class MMTFParser():
+
+    def get_structure_from_url(self, pdb_id):
+        """Get a structure from a URL - given a PDB id
```

Comment:
This is not a PEP257 style docstring - should be a single line, blank line, etc.

https://www.python.org/dev/peps/pep-0257/

I've not double checked this renders nicely as RST (reStructuredText) for our API docs.